src/styles/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid var(--color-gray);
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 40px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__description {
+  margin-bottom: 16px;
+}
+
+.card__title {
+  color: var(--color-primary);
+  font-weight: 500;
+  line-height: 18px;
+  font-style: normal;
+  font-size: 12px;
+  margin-bottom: 4px;
+}
+
+.card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  color: var(--color-primary);
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  font-style: normal;
+  color: var(--color-secondary);
+}
+
+.card__price-value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 14px;
+  font-style: normal;
+  color: var(--color-primary);
+  text-align: right;
+}
+
+.card__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 11px 69px;
```

Comment:
if you center the text why you need this padding?

Reply:
if I don't put padding. There is not enough background.